android/src/main/java/com/oney/WebRTCModule/WebRTCView.java

```diff
@@ -286,6 +293,13 @@ protected void onDetachedFromWindow() {
     private void onFirstFrameRendered() {
         post(() -> {
             Log.d(TAG, "First frame rendered.");
+            WritableMap event = Arguments.createMap();
+            ReactContext reactContext = (ReactContext) getContext();
+            reactContext.getJSModule(RCTEventEmitter.class).receiveEvent(
+                    getId(),
+                    "onFirstFrame",
+                    event
```

Comment:
can we skip the empty object here and pass null?

Reply:
Yes we can, fixed